src/hooks/useActiveDescendantListFocus/useActiveDescendantListFocusHooks.ts

```diff
@@ -100,14 +109,23 @@ export function useSupportPressItemKeyboardNavigation({
   isItemSelectable,
   listenerOptions = undefined,
   isIgnoreSpaceAsItemSelection = false
+}: {
+  visualFocusItemIndex: number;
+  focusedElementRef: MutableRefObject<HTMLElement>;
+  itemsCount: number;
+  setVisualFocusItemIndex: (index: number) => unknown;
+  onItemClick: (event: MouseEvent | KeyboardEvent, index: number) => unknown;
```

Comment:
if the type `MouseEvent | KeyboardEvent` is repeated a lot, consider extracting it to a common type